express-zod-api/src/worker.js

```diff
@@ -0,0 +1,27 @@
+import { clearInterval } from "node:timers";
+import { parentPort, workerData } from "node:worker_threads";
+
+if (!parentPort)
+  throw new Error("Logger worker must be run as a Worker Thread.");
+
+const { interval = 100 } = { ...workerData };
+
+/** @type string[] */
+const buffer = [];
+
+const flush = () => {
+  if (buffer.length === 0) return;
+  console.log(buffer.join("\n"));
+  buffer.length = 0;
+};
+
+/** @param {string} line */
+const onMessage = (line) => buffer.push(line);
+
+const job = setInterval(flush, interval);
+parentPort.on("message", onMessage);
+parentPort.on("close", () => {
```

Comment:
this does not happen. have to use `message` for all events

Reply:
fixed in [781e8ed](https://github.com/RobinTail/express-zod-api/pull/2497/commits/781e8ed2f0a6da8d8ce410450273137bfa236063)